lstchain/mc/sensitivity.py

```diff
@@ -768,10 +780,11 @@ def sensitivity(simtelfile_gammas, simtelfile_protons,
     # Build dataframe of events that survive the cuts:
     events = pd.concat((events_g, events_p))
     dl2 = pd.DataFrame(columns=events.keys())
+    
     for i in range(0, n_bins_energy):
         df_bin = events[(events.mc_energy < energy[i+1]) & (events.mc_energy > energy[i]) \
                                & (events.gammaness > gcut[i]) & (events.theta2 < tcut[i])]
 
         dl2 = pd.concat((dl2, df_bin))
 
-    return energy, sensitivity, result, units, dl2
+    return energy, sensitivity_flux, result, units, dl2
```

Comment:
the output being written in the `result` tuple is actually `sensitivity`, instead of `sensitivity_flux`. We can discuss about changing it, but I would not change it now because that will break some compatibility of people using this code

Reply:
OK I will keep sensitivity, but then I will add some lines in the lstchain_mc_sensitivity in order to plot the sensitivity_flux